packages/adapter-sqlite/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@lucia-auth/adapter-sqlite",
-	"version": "2.0.1",
+	"version": "3.0.0-beta.0",
 	"description": "SQLite adapter for Lucia",
 	"main": "dist/index.js",
 	"types": "dist/index.d.ts",
```

Comment:
While testing I had to deal with fixing the `peerDependencies` of this library. Any chance in the next beta we can point   the Lucia peer to `^3.0.0`?

Reply:
Ah thanks, never thought someone else was using the package

Reply:
No worries! I'm just playing around :)